charts/wuda-app/templates/ingress.yaml

```
@@ -35,9 +35,10 @@ spec:
             {{- end }}
             backend:
               service:
-                name: {{ include "wuda-app.fullname" $ }}
+                # kierujemy ruch na serwis frontendu, który nasłuchuje na porcie 8000
+                name: wuda-frontend
```

```suggestion
                name: {{ include "wuda-app.fullnameFrontend" . }}
```

Pretty sure it should be like this. And please, delete gpt comments. It looks bad.